src/dbnode/encoding/m3tsz/encoder.go

```diff
@@ -310,9 +310,26 @@ func (enc *encoder) LastEncoded() (ts.Datapoint, error) {
 	return result, nil
 }
 
-// Len returns the length of the data stream.
+// Len returns the length of the final data stream that would be generated
+// by a call to Stream().
```

Comment:
Hmm this is slightly redefining what `Len()` is returning, right? Are all other uses of this function okay with this new definition? Can you also change the comment for this function in its interface (in `dbnode/encoding/types.go`) if appropriate?

Reply:
Yeah this isn't used for much and all I've done is make it more accurate. Updated the comment